packages/openactive-integration-tests/documentation/generator.js

```diff
@@ -255,7 +255,7 @@ Opportunities that match the following criteria must exist in the booking system
 `)}
 ${renderTestInterfaceActionImplementationRequirements([...f.testInterfaceActionImplementationRequirements], `### Test prerequisites - Test Interface Actions
 
-The following Test Interface Actions must be implemented by the [test interface](https://openactive.io/test-interface/) of the booking system in order to test this feature:
+The following Test Interface Actions must be implemented by the [test interface](https://openactive.io/test-interface/) of the booking system in order to test this feature (see the [Developer Docs guide for implementing Test Interface Actions](https://developer.openactive.io/open-booking-api/test-suite/implementing-the-test-interface/test-interface-actions)):
```

Comment:
This link will be valid once Dev Docs gitbook PR https://app.gitbook.com/o/-LJTExWaXZFqcFJ_lZwF/s/-LOMz1IIOd5fl4xzfUYb/~/diff/~/changes/275/open-booking-api/test-suite/implementing-the-test-interface/test-interface-actions is merged